scripts/install.sh

```diff
@@ -61,7 +61,13 @@ if [ -n "${OS_TYPE}" ] && [ -n "${ARCH_TYPE}" ]; then
     fi
 
     echo "Downloading ${BIN}, OS: ${OS_TYPE}, Arch: ${ARCH_TYPE}, Version: ${VERSION}"
+    PACKAGE_NAME="${BIN}-${OS_TYPE}-${ARCH_TYPE}-${VERSION}.tar.gz"
+    wget "https://github.com/${GITHUB_ORG}/${GITHUB_REPO}/releases/download/${VERSION}/${PACKAGE_NAME}"
 
-    wget "https://github.com/${GITHUB_ORG}/${GITHUB_REPO}/releases/download/${VERSION}/${BIN}-${OS_TYPE}-${ARCH_TYPE}.tgz"
-    tar xvf ${BIN}-${OS_TYPE}-${ARCH_TYPE}.tgz && rm ${BIN}-${OS_TYPE}-${ARCH_TYPE}.tgz && echo "Run './${BIN} --help' to get started"
+    # Extract the binary and clean the rest.
+    tar xvf "${PACKAGE_NAME}" && \
+    mv "${PACKAGE_NAME%.tar.gz}/${BIN}" "${PWD}" && \
+    rm -r "${PACKAGE_NAME}" && \
```

Comment:
this seems dangerous if PACKAGE_NAME is accidentally empty

Reply:
you are VERY right!!!

Reply:
When I review the code again, I found that:

- `${BIN}` already got the assigned value `greptime` and cannot be empty;
- `${OS_TYPE}`, `${ARCH_TYPE}` and `${VERSION}` is already checked before `rm` operation;

So, the `PACKAGE_NAME` cannot be empty. However, I still add the guard check to make sure `PACKAGE_NAME` cannot be empty.

Reply:
Yes. But we need to prevent future refactoring that changes how PACKAGE_NAME is assign.